src/pages/Support.svelte

```diff
@@ -33,7 +33,7 @@
 
 		or<br /><br />
 
-		<a href="mailto:golova@golova.dev">Write me an email</a>. And I'll know a <a href="/clan/GENX">clan for you</a> ¯\_(ツ)_/¯<br /><br />
+		<a href="mailto:golova@golova.dev">Write me an email</a>. And I'll know a <a href="/clan/GENX/players/1?">clan for you</a> ¯\_(ツ)_/¯<br /><br />
```

Comment:
Fixed, a few months ago, I had the idea to put a clanRankings tab on each clan page. I cut that code out because I think it should be in a later PR now. But I missed this while ripping things out.